integration/build.gradle

```diff
@@ -3,6 +3,29 @@ repositories {
     maven { url 'http://repository.jboss.org/nexus/content/groups/public' }
 }
 
+project.configurations {
+    integration {
+        extendsFrom project.configurations.testRuntime
+        description = 'Dependencies for integration tests'
+        transitive = true
+        visible = true
+    }
+    alpnboot
+}
+
+def getAlpnVersion() {
```

Comment:
how important is this? can we just skip it somehow and assuem we use newest jvms? (check on CI)